client/my-sites/people/delete-user/index.jsx

```diff
@@ -246,7 +246,7 @@ class DeleteUser extends React.PureComponent {
 					</FormFieldset>
 
 					<FormButtonsBar>
-						<FormButton disabled={ this.isDeleteButtonDisabled() }>
+						<FormButton isScary={ true } disabled={ this.isDeleteButtonDisabled() }>
```

Comment:
All props are passed through to `<Button />` already by the line `{ ...omit( props, ...` in `<FormButton />`, so if you make this `scary={ true }`, you won't need the form-button changes.

Reply:
Thank you for catching that. Next time I'll test it out bit by bit without assuming and overthinking. :D